Eastern-Research-Group/Trade-off_Tool_for_Sampling
# Feature/56 hybrid mode

## Related Issues:
* [TOTS-56](https://ergcloud.atlassian.net/browse/TOTS-56)

## Main Changes:
* Added in a hybrid shape mode. This one makes it so points are displayed as points and polygons are displayed as polygons.
* Added in a test for the new hybrid shape mode.
* Renamed the cypress related files to consistently use `-`. Before some files used `-` and some used `_`.

## Steps To Test:
1. Add some samples to the map with a mix of sample types
2. Switch between points view, polygons view, and hybrid view. Verify they display appropriately
    * Points should display all sample types as points.
    * Polygons should display all sample types as polygons.
    * Hybrid should display point based sample types as points and polygon based sample types as polygons.
3. Run cypress and verify tests pass
